CONTRIBUTING.md

```diff
@@ -225,7 +225,7 @@ Instead of directly querying the GraphQL, you can also run `distinct` or `regex`
 
 4. You can now query you local MongoDB:
 
-	```
+	```go
```

Comment:
This is `sh` then a mongo shell so if there's no mongo syntaxing I would just leave it plain.

Reply:
Oops, sorry, should revert it here. Just looking around and see how to contribute to codes, good to know that I found something I understood wrongly before actually doing something 😅

Reply:
@aliscott should here be `mongosh` instead of `mongo`?

According to [MongoDB Shell](https://docs.mongodb.com/mongodb-shell/):

> The MongoDB Shell, mongosh, is a fully functional JavaScript and Node.js 14.x REPL environment for interacting with MongoDB deployments. You can use the MongoDB Shell to test queries and operations directly with your database.

I think we can use `javascript` as its language.

Reply:
Yeah I think `javascript` should work, I don't think GitHub markdown supports anything special for mongo.

Reply:
@aliscott so should here be `mongosh` instead of `mongo`? Maybe should also separate the shell command from the code block.

Reply:
I think just `javascript` is fine. Here's an example of how the syntax highlighting will look below:

```javascript
mongo
use cloudPricing;

// Find the vendor names
db.products.distinct("vendorName");
// should show: [ "aws", "azure", "gcp" ]

// Find all the services for a vendor
db.products.distinct("service", {"vendorName": "gcp"})

// Find the service for a vendor using a regular expression
db.products.distinct("service", {"vendorName": "gcp", "service": {$regex: "kms", $options: "i"} });
// should show: [ "Cloud Key Management Service (KMS)", "Thales CPL ekms-dpod-eu" ]

// Find the product families for a service
db.products.distinct("productFamily", {"vendorName": "gcp", "service": "Cloud Key Management Service (KMS)"})

// Find the unique descriptions for a product family
db.products.distinct("attributes.description", {"vendorName": "gcp", "service": "Cloud Key Management Service (KMS)", "productFamily": "ApplicationServices" });
// should show:  [
//	"Active HSM ECDSA P-256 key versions",
//	"Active HSM ECDSA P-384 key versions",
//	"Active HSM RSA 2048 bit key versions",
//  ...
// ]

// Find a unique product for a product family based on region and description:
db.products.find({"vendorName": "gcp", "service": "Cloud Key Management Service (KMS)", "productFamily": "ApplicationServices", "region": "us-east1", "attributes.description": "Active HSM ECDSA P-256 key versions" }).pretty()
```

Reply:
@aliscott I mean that you say
> This is `sh` then a mongo shell

Does that mean the first `mongo` a command under shell? If so, should better be separated?

Reply:
Yes the first line `mongo` is run under a shell so we could separate it like this:

Run the mongo shell:

```sh
mongo
```

Example queries:

```javascript
use cloudPricing;

// Find the vendor names
db.products.distinct("vendorName");
// should show: [ "aws", "azure", "gcp" ]

// Find all the services for a vendor
db.products.distinct("service", {"vendorName": "gcp"})

// Find the service for a vendor using a regular expression
db.products.distinct("service", {"vendorName": "gcp", "service": {$regex: "kms", $options: "i"} });
// should show: [ "Cloud Key Management Service (KMS)", "Thales CPL ekms-dpod-eu" ]

// Find the product families for a service
db.products.distinct("productFamily", {"vendorName": "gcp", "service": "Cloud Key Management Service (KMS)"})

// Find the unique descriptions for a product family
db.products.distinct("attributes.description", {"vendorName": "gcp", "service": "Cloud Key Management Service (KMS)", "productFamily": "ApplicationServices" });
// should show:  [
//	"Active HSM ECDSA P-256 key versions",
//	"Active HSM ECDSA P-384 key versions",
//	"Active HSM RSA 2048 bit key versions",
//  ...
// ]

// Find a unique product for a product family based on region and description:
db.products.find({"vendorName": "gcp", "service": "Cloud Key Management Service (KMS)", "productFamily": "ApplicationServices", "region": "us-east1", "attributes.description": "Active HSM ECDSA P-256 key versions" }).pretty()
```

Reply:
Cool, then I think we're on the same page! I'll update the commit and resolve the conflicts now, looks like there are new commits over the file a few hours before.

Reply:
@aliscott should be done now, please take a look, and thanks for the review.

Reply:
Perfect thank you @PeterDaveHello!

Reply:
oops, we missed a question here, shouldn't we use `mongosh` instead of `mongo` for the shell commend here?